presto-native-execution/scripts/setup-adapters.sh

```diff
@@ -35,15 +35,73 @@ function install_prometheus_cpp {
   cmake_install -DBUILD_SHARED_LIBS=ON -DENABLE_PUSH=OFF -DENABLE_COMPRESSION=OFF
 }
 
+function install_abseil {
+  # abseil-cpp
+  github_checkout abseil/abseil-cpp 20240116.2 --depth 1
+  cmake_install \
+    -DABSL_BUILD_TESTING=OFF \
+    -DCMAKE_CXX_STANDARD=17 \
+    -DABSL_PROPAGATE_CXX_STD=ON \
+    -DABSL_ENABLE_INSTALL=ON
+}
+
+function install_grpc {
+  # grpc
+  github_checkout grpc/grpc v1.48.1 --depth 1
+  cmake_install \
+    -DgRPC_BUILD_TESTS=OFF \
+    -DgRPC_ABSL_PROVIDER=package \
+    -DgRPC_ZLIB_PROVIDER=package \
+    -DgRPC_CARES_PROVIDER=package \
+    -DgRPC_RE2_PROVIDER=package \
+    -DgRPC_SSL_PROVIDER=package \
+    -DgRPC_PROTOBUF_PROVIDER=package \
+    -DgRPC_INSTALL=ON
+}
+
+function install_arrow_flight {
+  ARROW_VERSION="15.0.0"
```

Comment:
We should get this from the Velox script. We can skip here to align the versions.

Reply:
@majetideepak the version is defined in the OS setup scripts, we could do something like
```
ARROW_VERSION="${ARROW_VERSION:-15.0.0}"
```